proposals/spi-proposal.md

```diff
@@ -52,9 +51,9 @@ has multiple ports (SPI, SPI1, SPI2, etc). Many only have one port so the
 
 ```
 0:  START_SYSEX
-1:  SPI_DATA             (0x68)
-2:  SPI_CONFIG           (0x00)
-3:  channel              (HW supports multiple SPI ports. range = 0-3, default = 0)
+1:  SPI_DATA              (0x68)
+2:  SPI_CONFIG            (0x00)
```

Comment:
Could get rid of `SPI_CONFIG` and handle it implicitly the first time calling `SPI_BEGIN_TRANSFER` for a channel.